exercism/python/pangram/pangram.py

```diff
@@ -1,3 +1,111 @@
+# Initial Solution - Use a string
+from string import ascii_lowercase as lowercase_letters
+
+
+def is_pangram(sentence):
+    letters = ""
+
+    for char in sentence.lower():
+        if char.isalpha() and char not in letters:
+            letters += char
+    return len(letters) == len(lowercase_letters)
+
+"""
+Pros: Simple to read and understand
+Cons: Not efficient in terms of complexity. That's because strings are immutable in Python. Each time I concatentate strings, an entirely new string is created and the letter is added to it.
+"""
+
+
+###Solutions When Apply Looking Back Approaches###
+
+# Approach 1: Solve problem with a different approach
+# Approach 1 Solution - ASCII representation
+from string import ascii_lowercase as lowercase_letters
+
+ALPHA_LENGTH = 26
```

Comment:
Instead of making this constant, you could alternatively just use `len(lowercase_letters)` in the two spots in the code.